CHANGELOG.md

```diff
@@ -7,3 +7,7 @@ This project uses [CalVer][calver] - YY.M.D.MICRO
 - NSX functionality
 - VMC functionality
 - ESXi functionality
+
+# [22.11.18]
```

Comment:
**suggestion** this should be in the changelog/NNN.added - if there was no issue created here in the repo can just be 323.added (and it should say "Added vmware_esxi.set_ntp_config module" probably)